subsys/net/ip/net_shell.c

```diff
@@ -3535,7 +3535,347 @@ static int cmd_net_ipv6(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
-static int cmd_net_iface(const struct shell *sh, size_t argc, char *argv[])
+static int cmd_net_ip6_add(const struct shell *shell, size_t argc, char *argv[])
```

Comment:
CI complains about using `shell` as the parameter name, please use `sh`:
> subsys/net/ip/net_shell.c:3577:WARNING: Violation to rule 5.7 (Tag name should be unique) tag: shell

Reply:
Done